pyincore_viz/analysis/popresultstable.py

```diff
@@ -261,6 +261,11 @@ def pop_results_table(df, **kwargs):
         Returns:
             object: Styled Table Pandas DataFrame object.
         """
+
+        # Create copy of input dataframe
+        # Copy prevents original dataframe from being altered
+        df = input_df.copy()
+
         who = ""
         what = ""
         when = ""
```

Comment:
seems like who/what/when/where are fixed input parameters names? Do you think we should explicitly include them rather then just pass in kwargs?

Reply:
Nathanael made this way and I can ask him if he wants. But if I recall correctly, I guess that I have asked this already and he said it was okay to do this way because there will not be any more for this.